changelog.d/5543.misc

```diff
@@ -0,0 +1 @@
+Make the config clearer in that email.template_dir is relative to the Synapse's root directory, not the `synapse/` folder within it.
```

Comment:
I don't think this is correct

Reply:
True, it's relative to the virtualenv, and that'll require some more detailed explanation in a comment.

> when we did the email thing there was no indication to the admins that they had to had to a new template to it.

Right, they'll need a new template if they set a custom template directory. I'll need to update https://github.com/matrix-org/synapse/pull/5545 then.

Reply:
Actually, I think the errors should be clear enough for this usecase.